sdk-ext-lib/src/main/java/cash/z/ecc/sdk/fixture/WalletAddressFixture.kt

```diff
@@ -6,13 +6,14 @@ object WalletAddressFixture {
     // These fixture values are derived from the secret defined in PersistableWalletFixture
 
     // TODO [#161]: Pending SDK support
+    // TODO [#161]: https://github.com/zcash/secant-android-wallet/issues/161
     const val UNIFIED_ADDRESS_STRING = "Unified GitHub Issue #161"
 
     @Suppress("MaxLineLength")
-    const val SHIELDED_SAPLING_ADDRESS_STRING = "ztestsapling1475xtm56czrzmleqzzlu4cxvjjfsy2p6rv78q07232cpsx5ee52k0mn5jyndq09mampkgvrxnwg"
-    const val TRANSPARENT_ADDRESS_STRING = "tmXuTnE11JojToagTqxXUn6KvdxDE3iLKbp"
+    const val LEGACY_SAPLING_ADDRESS_STRING = "ztestsapling1475xtm56czrzmleqzzlu4cxvjjfsy2p6rv78q07232cpsx5ee52k0mn5jyndq09mampkgvrxnwg"
+    const val LEGACY_TRANSPARENT_ADDRESS_STRING = "tmXuTnE11JojToagTqxXUn6KvdxDE3iLKbp"
```

Comment:
Do we need to regenerate the transparent address string?  Not sure which method was used to derive it, so it might be different.